<launch>

<include file="$(find ucar_nav)/launch/navi_fast.launch" > </include> 

<!-- <node pkg="ucar_cam" name="ucar_cam" type="usb_camera_SR.py" output="screen"/>
 
<node pkg="sr_pkg" name="darknet_node" type="darknet_node_SR.py" output="screen"/> -->

<!-- <node pkg="sr_pkg" name="smartcar_test" type="SR_nav_02.py" output="screen"/> -->


 <!-- <node pkg="ucar_cam" name="ucar_cam" type="usb_camera_SR.py" output="screen"/>

 <node pkg="sr_pkg" name="darknet_node" type="darknet_node_SR.py" output="screen"/> -->

<node name="rviz" pkg="rviz" type="rviz" args="-d $(find sr_pkg)/launch/ucar.rviz"/>

</launch>
